src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs

```diff
@@ -470,6 +461,22 @@ protected override void Dispose(bool disposing)
         return bodyLength;
     }
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    private void SerializeHistogramBucket(HistogramBucket bucket, ref int bufferIndex, double lastExplicitBound)
```

Comment:
@CodeBlanch Should we add the `in` parameter modifier for `HistogramBucket` as it's a readonly struct?

I'm not sure if it would be required considering we use `AggressiveInlining`.

Reply:
I messed with it a bit on SharpLab the JIT seems smart enough to do the right thing but it wouldn't hurt anything to add `in` on there I don't think.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/842